lib/widgets/navigation_drawer.dart

```diff
@@ -18,9 +18,14 @@ class DrawerItemConfig {
   final String icon;
   final bool disabled;
   final void Function(String name) onItemSelected;
+  final void Function() onToggle;
+  final toggleValue;
 
   DrawerItemConfig(this.name, this.title, this.icon,
-      {this.onItemSelected, this.disabled = false});
+      {this.onItemSelected,
+      this.disabled = false,
+      this.onToggle,
```

Comment:
Isn't it better to pass a builder function (that builds widget) here instead of _onToggle_ & _toggleValue_?

Reply:
[You are right](https://github.com/breez/breezmobile/pull/277/commits/cabf313c64cff64ba70ac765fdacd7d132f217ba)